ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -522,6 +526,12 @@ final class Metrics(val registry: MetricRegistry) {
       )
 
       val stateUpdateProcessing: Timer = registry.timer(Prefix :+ "processed_state_updates")
+
+      val currentStateCacheSequentialIdGauge = new VarGauge[Long](0L)
+      registry.register(
+        Prefix :+ "current_state_cache_sequential_id",
```

Comment:
AFAIS, we do not have its counterpart `ledger_end_sequential_id` that would allow us to compute the lag. What do you think about adding that?

Reply:
Good point. Noted down to implement in the harden metrics task.